directembedding/src/main/scala/ch/epfl/directembedding/transformers/DSLVirtualization.scala

```diff
@@ -0,0 +1,23 @@
+package ch.epfl.directembedding.transformers
+
+import ch.epfl.directembedding.{ DirectEmbeddingUtils, DirectEmbeddingModule }
+import ch.epfl.yinyang.EmbeddedControls
+
+object RewireEmbeddedControls extends EmbeddedControls
+
+trait DSLVirtualization extends DirectEmbeddingModule with DirectEmbeddingUtils {
+  import c.universe._
+
+  def DSLVirtualizer = new (Tree => Tree) {
+    def apply(tree: Tree) = {
+      c.typecheck(q"""
+      import _root_.ch.epfl.directembedding.transformers.RewireEmbeddedControls._
+      {
+          import $configPath._
```

Comment:
We import everthing from the configuration type, this guarantees that the `lift` method is no scope after the transformation (needed separate import before).

The caveat is that the language virtualization overrides must now be part of the same object as the configuration, as seen [here](https://github.com/directembedding/directembedding/pull/23/files#diff-d1429d3c359afa93eb64775fe183f5c8R30)